policy/types.go

```diff
@@ -365,3 +367,23 @@ type RuntimeError struct {
 	ContextID string
 	Error     error
 }
+
+// Fnv32Hash hash the given data by Fnv32-bit algorithm.
+func Fnv32Hash(data ...string) (string, error) {
+
+	if len(data) == 0 {
+		return "", fmt.Errorf("no data to hash")
+	}
+
+	aggregatedData := ""
+	for _, ed := range data {
+		aggregatedData += ed
+	}
+
+	hash := fnv.New32()
+	if _, err := hash.Write([]byte(aggregatedData)); err != nil {
+		return "", fmt.Errorf("unable to hash data: %v", err)
+	}
+
+	return fmt.Sprintf("%d", hash.Sum32()), nil
+}
```

Comment:
Confused. Who is using this method? I don't see anything below.

Reply:
This is used by this PR https://github.com/aporeto-inc/trireme-lib/pull/838 and as well as in enforcer which is for a a different use case. Which ever goes first the other will either be closed or conflicts will be resolved.